src/main.ts

```diff
@@ -546,15 +552,30 @@ export class Purchases {
 
     const localeToBeUsed = selectedLocale || defaultLocale;
 
+    const purchaseOptionToUse =
+      purchaseOption ?? rcPackage.rcBillingProduct.defaultPurchaseOption;
```

Comment:
the `rcBillingProduct` property was recently marked as deprecated. It's still there but might change in the future.
```suggestion
      purchaseOption ?? rcPackage.webBillingProduct.defaultPurchaseOption;
```

Reply:
The `user-events` branch is now a bit behind so `webBillingProduct` does not exist here. I will look into this once I rebase `user-events` from `main` or merge `main` into `user-events`.